website/content/getting-started.md

```diff
@@ -1,11 +1,11 @@
 ---
 {
-  "title": "Home",
-  "date": "2020-07-06T00:00:00",
-  "author": "Felix Queißner",
-  "draft": false,
-  "layout": "getting-started.html",
-  "tags": []
+  .title = "Home",
+  .date = "2020-07-06T00:00:00",
```

Comment:
Does this build without the `@date(..)`?

Reply:
It does build but it's not correct. I'll fix them up.